### PR TITLE
RWA-2485: Available Tasks / My Tasks issue after POC deployed

### PR DESCRIPTION
**Pull request by rasikasri**
RWA-2486: Error in logs following performance improvements deployment

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/resources/db/migration/V1.0.26__update_performance_index_function.sql [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+$$;
+
+/*
+ * GIN index to look up tasks based on role signatures and filter signatures.
+ * Only covers assigned and unassigned tasks which have been flagged as
+ * indexed.  There is no value in indexing old tasks, since we never search
+ * them.
+ */
+drop index if exists cft_task_db.search_index;
+create index search_index on cft_task_db.tasks using gin (
```
Comment (raghera) [inline src/main/resources/db/migration/V1.0.26__update_performance_index_function.sql:156]: Any way we could reliably reproduce the problem using a functional test or only possible directly with the DB with fill factor etc.

**Review by raghera — COMMENTED** [review]
The performance fix looks good but a couple of questions.

File: src/main/resources/db/migration/V1.0.26__update_performance_index_function.sql (hunk 1 above)
Comment (rasikasri) [reply src/main/resources/db/migration/V1.0.26__update_performance_index_function.sql:156]: Well, We arealy have a functional test that doing the exact same steps, but that didn't have the issue.

File: src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/search/SortField.java [hunk 2]
```diff
@@ -10,8 +10,8 @@ public enum SortField {
     DUE_DATE_CAMEL_CASE("dueDate", "due_date_time", "dueDateTime"),
     DUE_DATE_SNAKE_CASE("due_date", "due_date_time", "dueDateTime"),
 
-    TASK_TITLE_CAMEL_CASE("taskTitle", "task_title", "title"),
-    TASK_TITLE_SNAKE_CASE("task_title", "task_title", "title"),
+    TASK_TITLE_CAMEL_CASE("taskTitle", "title", "title"),
```
[the remark this answers is not in the document]
Comment (rasikasri) [reply src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/search/SortField.java:13]: This doesn't change neither. I only corrected the db filed name from task_title to title. It is not visible to outside.

**Review by dharmendrak — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/search/SortField.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (raghera) [reply src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/search/SortField.java:13]: Ok should be fine then

**Pull request merged**